.ci/code/BHoM_Adapter_Tests/Objects/StructuralAdapter.cs

```diff
@@ -75,8 +76,20 @@ public StructuralAdapter()
                 { typeof(GravityLoad), new List<Type>{ typeof(Bar), typeof(Panel), typeof(FEMesh)} }
             };
 
+            AdapterComparers = new Dictionary<Type, object>
+            {
+                {typeof(Bar), new BarEndNodesDistanceComparer(3) },
+                {typeof(Node), new NodeDistanceComparer(3) },
+                {typeof(ISectionProperty), new NameOrDescriptionComparer() },
+                {typeof(ISurfaceProperty), new NameOrDescriptionComparer() },
+                {typeof(IMaterialFragment), new NameOrDescriptionComparer() },
+                {typeof(LinkConstraint), new NameOrDescriptionComparer() },
+                {typeof(Constraint6DOF), new NameOrDescriptionComparer() },
+            };
+
             AdapterIdFragmentType = typeof(StructuralAdapterId);
             BH.Adapter.Modules.Structure.ModuleLoader.LoadModules(this);
+            this.m_AdapterSettings.OnlyUpdateChangedObjects = true;
```

Comment:
Could we move this on line 59 (i.e. inside the Adaptersetting initialisation)?

Reply:
Sure

Reply:
Actually missed the AdapterSettings constructor above, fixed now :)